docs/src/examples/components/Icon/States/IconExampleDisabled.shorthand.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Icon } from '@stardust-ui/react'
+
+const IconExampleDisabled = () => (
+  <div>
+    <Icon disabled color="red" name="users" size="big" />
```

Comment:
Let's also have a non-colored disabled icon here for completeness.

Reply:
added